packages/belos/tpetra/example/WrapTpetraSolver/wrapTpetraSolver.cpp

```diff
@@ -544,6 +544,10 @@ class CgWrapper :
   Belos::ReturnType solve () {
     TEUCHOS_TEST_FOR_EXCEPTION(true, std::logic_error, "Not implemented");
   }
+
+  virtual Teuchos::RCP<Belos::SolverManager<ScalarType, MV, OP> > clone () const {
+    TEUCHOS_TEST_FOR_EXCEPTION(true, std::logic_error, "clone() not implemented");
```

Comment:
Why not?

Reply:
This `clone` approach is actually a nice replacement or alternative for the "custom solver factory" approach that this test exercises.

Reply:
I added the clone method to call the construction of this class. However there is no test requirement yet for this to be implemented. I hadn't looked into the details of these classes yet.